nautobot/docs/development/core/local-k8s.md

```diff
@@ -0,0 +1,126 @@
+# Using Local Kubernetes Cluster
+
+This guide will help you set up a local Kubernetes cluster using [Kind](https://kind.sigs.k8s.io/) and connect running Nautobot containers to the Kind network.
+
+## Kind Installation
+
+To install Kind on macOS:
+
+```bash
+brew install kind
+```
+
+To install Kind on Linux:
+
+```bash
+export KIND_VERSION=0.24.0
+curl -fsSL \
+    "https://kind.sigs.k8s.io/dl/v$KIND_VERSION/kind-linux-amd64" \
+    -o /usr/local/bin/kind
+chmod +x /usr/local/bin/kind
+```
+
+To create a new Kind cluster:
+
+```bash
+# Create the cluster
+kind create cluster \
+    --name=nautobot \
+    --config development/kind-config.yaml
+
+# Export the kubeconfig to access the cluster
+kind get kubeconfig --internal --name=nautobot > development/kind-kube-config
+```
+
+## Connecting Nautobot Containers
+
+This chapter will help you connect running Nautobot containers to the Kind network. It is necessary to do this after starting the Nautobot containers.
+
+Verify Nautobot containers are running:
+
+```bash
+# Start Nautobot containers if they are not running
+invoke start
+
+# Verify the containers are running
+docker ps
+```
+
+You should see something like this:
```

Comment:
What in particular should the reader be looking for? Presumably the fact that `nautobot-nautobot-1` and `nautobot-celery-worker-1` both show as `(healthy)`? Would be good to be explicit for new readers.

Reply:
Added